.github/workflows/publish-release.yml

```diff
@@ -1,28 +1,36 @@
 name: Publish Release
 
 on:
-  pull_request:
-    types: [closed]
+  push:
+    types: [main]
 
 jobs:
   publish-release:
     permissions:
       contents: write
-    if: |
-      github.event.pull_request.merged == true &&
-      startsWith(github.event.pull_request.head.ref, 'release/')
+    # release merge commits come from GitHub user
+    if: github.event.head_commit.committer.name == 'GitHub'
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
         with:
           ref: ${{ github.sha }}
+          # we need this commit + the last so we can compare below
+          fetch-depth: 2
+      # exit early if the version has not changed
+      - run: |
+          VERSION_BEFORE="$(git show ${{ github.event.before }}:package.json | jq --raw-output .version)"
+          VERSION_AFTER="$(jq --raw-output .version package.json)"
+          if [[ "$VERSION_BEFORE" == "$VERSION_AFTER" ]]; then
+            exit 0
+          fi
       - name: Get Node.js version
         id: nvm
         run: echo ::set-output name=NODE_VERSION::$(cat .nvmrc)
       - uses: actions/setup-node@v2
         with:
           node-version: ${{ steps.nvm.outputs.NODE_VERSION }}
-      - uses: MetaMask/action-publish-release@v1
+      - uses: MetaMask/action-publish-release@v2
```

Comment:
this needs to be merged: https://github.com/MetaMask/action-publish-release/pull/46 and then a new version released before this will work.

Reply:
@rickycodes Hope you're feeling better. Just a reminder that this is merged now but a release still needs to be cut.